lib/helper/import.js

```diff
@@ -0,0 +1,100 @@
+'use babel';
+
+const atom = global.atom;
+const Path = require('path');
+const FileSystem = require('fs-plus');
+const Electron = require('electron');
+const Storage = require('./../helper/storage.js');
+const XMLParser = require('xml-js');
+const server_config = require('./../config/server-schema.json');
+const folder_config = require('./../config/folder-schema.json');
+
+class Import {
+  constructor() {
+    const self = this;
+  }
+
+  import(configuration_view) {
+    const self = this;
+    Electron.remote.dialog.showOpenDialog(null, { title: 'Select file to import', defaultPath: Electron.remote.app.getPath("desktop"), buttonLabel: 'Import', filters: [{ name: 'All Files', extensions: ['*'] }, { name: 'remote-ftp config file', extensions: ['ftpconfig'] }, { name: 'FileZilla site export file', extensions: ['xml'] }], properties: ['openFile', 'showHiddenFiles'] }, (filePaths, bookmarks) => {
+      if (filePaths) {
+        filePaths.forEach((filePath, index) => {
+          if (Path.extname(filePath).toLowerCase() == '.xml') {
+            FileSystem.readFile(filePath, function (err, data) {
+              var json = XMLParser.xml2js(data, { compact: true });
+              if (typeof json['FileZilla3'] !== 'undefined') {
+                self.importFileZilla3(json['FileZilla3']['Servers']['Folder'], null);
```

Comment:
If the filezilla configuration does not contain any folders, the server list starts with the server instead of the folder.

Reply:
So removed ['Folder'] from here, rest of code will handle it.